### PR TITLE
DCOS_OOS-718: Fix scrollTop of undefined when resizing logs container

### PR DESCRIPTION
**Pull request by nLight**
I couldn't help refactoring a little. I find it's better to have explicit components when we are expecting them as a test result.

This PR addresses hopefully all the something of undefined issues caused by not having a container ref before we use it.

And one more thing: for me go down button will stay on the empty screen so I moved it under the condition as well.

Feel free to drop/rebase -i commits

:bow:

**Checklist**
- [ ] Did you add a JIRA issue in a commit message or as part of the branch name?
- [ ] Did you add new unit tests?
- [ ] Did you add new integration tests?
- [ ] If this is a regression, did you write a test to catch this in the future?

### CODE REVIEW

File: plugins/services/src/js/components/__tests__/MesosLogView-test.js [hunk 1]
```diff
@@ -178,14 +179,14 @@ describe('MesosLogView', function () {
 
     it('should show empty log when fullLog is empty string', function () {
       this.instance.state.fullLog = '';
-      var div = this.instance.getLog();
-      expect(TestUtils.isElementOfType(div, 'div')).toEqual(true);
+      var res = this.instance.getLog();
+      expect(TestUtils.isElementOfType(res, EmptyLogScreen)).toEqual(true);
```
Comment (orlandohohmeier) [inline plugins/services/src/js/components/__tests__/MesosLogView-test.js]: ❤️

**Comment by orlandohohmeier** [comment]
**mesosphere-ci** retest this please 🙏

File: plugins/services/src/js/components/EmptyLogScreen.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        {`${logName} Log is Currently Empty`}
+      </h3>
+      <p className="text-align-center flush-bottom">
+        Please try again later.
+      </p>
+    </div>
+  );
+};
+
+module.exports = EmptyLogScreen;
```
Comment (MatApple) [inline plugins/services/src/js/components/EmptyLogScreen.js]: Let's add a propType for logName!

File: plugins/services/src/js/components/__tests__/MesosLogView-test.js (hunk 1 above)
Comment (MatApple) [inline plugins/services/src/js/components/__tests__/MesosLogView-test.js]: haha 👍

File: plugins/services/src/js/components/EmptyLogScreen.js (hunk 2 above)
Comment (rcorral) [inline plugins/services/src/js/components/EmptyLogScreen.js]: Functional stateless components don't have a way of defining proptypes.

Comment (MatApple) [inline plugins/services/src/js/components/EmptyLogScreen.js]: About to push the change

Comment (MatApple) [inline plugins/services/src/js/components/EmptyLogScreen.js]: @rcorral https://facebook.github.io/react/blog/2015/10/07/react-v0.14.html#stateless-functional-components

Comment (rcorral) [inline plugins/services/src/js/components/EmptyLogScreen.js]: Oh nice, I didn't know this. Thanks!

**Comment by rcorral** [comment]
Looks good to me too!

**Pull request merged**